develop/devguide/Connector/ImpactOfProtocolChanges/TableParameterChanges/ChangeDisplayKey.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: ChangeDisplayKey
+---
+
+# Change display key
+
+Changing the display key of a table is considered a major change.
+
+## Impact
+
+This has the same impact as changing a parameter description. The following DataMiner features can be impacted if used:
+
+- Filters
+- Automation scripts
+- Visio files
+- Reports
+- Web API
+
+*DIS MCC*
+
+|            | Error message | Description                                                                                                        |
+|------------|---------------|--------------------------------------------------------------------------------------------------------------------|
+| **2.39.4** | FormatChanged | Table display key was changed from {oldSyntax} '{oldFormat}' to {newSyntax} '{newFormat}'. Table PID '{tablePid}'. |
+| **2.39.5** | FormatRemoved | Table display key previously defined via '{oldSyntax}' was removed. Table PID '{tablePid}'.                        |
+
+## Workarounds
+
+### Display key column exists
+
+If a display key already exists, and you want to change its format, the following routine should be implemented to avoid impact by default:
+
+- R/W (standalone) discreet parameter to select the naming format
+
+- Default value should be the original format (no impact for existing users)
+
+- Ref. SLC SDF Naming Format Compatibility
```

Comment:
This is not clear, and the SVN link below does not work. How should this be adjusted?

Reply:
Here is the SVN link: "https://svn.skyline.be/svn/SystemDevelopmentFeature/Protocols/SLC SDF Naming Format Compatibility". That's only available to Skyliners though. One day, we want to move such examples to github but we shouldn't be waiting on that to release these new pages. Also note that, we want to rephrase things a bit here so that we refer to the fact the best way to tackle this would be to have a new software feature allowing us to define multiple display key formats. It's only in the meantime and when there is no other option that we should go for the workaround of adding a configurable display key format and building it ourselves via QActions.

But again, and this holds for most of the remarks on this PR, I think it's ok to first release a not perfect version. (this info is now already gone from the old wiki and colleagues are seeking for it). We can improve this iteratively afterwards. We however should take notes of all the remarks you provided Marieke so we can eventually look into improving those things.